common/protobuf/Tuning.proto

```diff
@@ -0,0 +1,8 @@
+package Packet;
```

Comment:
Let's leave the tuning-related things out of this PR and wait until the next PR.